apps/website/src/pages/collaborations.tsx

```diff
@@ -219,7 +227,7 @@ const CollaborationItems = forwardRef<
         <Heading
           level={-1}
           className={classes(
-            "alveus-green-800 mb-6 mt-8 border-b-2 border-alveus-green-300/25 pb-2 text-4xl",
+            "alveus-green-800 mb-6 mt-8 scroll-mt-16 border-b-2 border-alveus-green-300/25 pb-2 text-4xl",
             index === 0 && "sr-only",
```

Comment:
For the first link we're using `sr-only` which means that the `scroll-mt-*` doesn't work.

I think we'll need to write a custom Tailwind utility class to get around this, something like:

```scss
@layer utilities {
  .sr-only-linkable {
    @apply sr-only relative m-0 p-0 border-0;
    
    font-size: 0;
    line-height: 0;
  }
}
```